### ISSUE TITLE
Uninstall?

### ISSUE THREAD

**Issue by switz** [statianzo/dokku-supervisord#25]
I tried to remove it from the plugins directory, but it broke my apps:

```
2015-01-14 21:12:01,500 DEBG fd 8 closed, stopped monitoring <POutputDispatcher at 140396935297648 for <Subprocess at 140396935215008 with name web in state STARTING> (stderr)>
2015-01-14 21:12:01,500 DEBG fd 6 closed, stopped monitoring <POutputDispatcher at 140396935215152 for <Subprocess at 140396935215008 with name web in state STARTING> (stdout)>
2015-01-14 21:12:01,501 INFO exited: web (exit status 1; not expected)
2015-01-14 21:12:01,501 DEBG received SIGCLD indicating a child quit
2015-01-14 21:12:04,507 INFO spawned: 'web' with pid 25
2015-01-14 21:12:04,764 DEBG 'web' stderr output:
chown:
2015-01-14 21:12:04,764 DEBG 'web' stderr output:
changing ownership of '/app/supervisord.conf'
2015-01-14 21:12:04,765 DEBG 'web' stderr output:
: Operation not permitted
2015-01-14 21:12:04,766 DEBG 'web' stderr output:
```

**Comment by kenips** [comment]
You'll need to redeploy you app to remove it.

**Comment by kenips** [comment]
This shouldn't be a bug after uninstalling. And this bug itself is also fixed by #26. So please reopen if you have any new issues. Thanks!
